src/ResourceManager/Sql/Commands.Sql/ServiceObjective/Cmdlet/AzureSqlServerServiceObjectiveCmdletBase.cs

```diff
@@ -47,6 +47,7 @@ public abstract class AzureSqlServerServiceObjectiveCmdletBase
             HelpMessage = "SQL Database name.",
             DontShow = true)]
         [ValidateNotNullOrEmpty]
+        [Obsolete("This parameter is not needed or used. It will be removed in a future release.")]
```

Comment:
If you plan to do this in the next breaking change release, you can update this to say "in the next breaking change release".

Reply:
done.